apps/antalmanac/src/stores/calendarizeHelpers.ts

```diff
@@ -223,3 +223,20 @@ export function translate24To12HourTime(startTime?: HourMinute, endTime?: HourMi
 
     return `${meetingStartTime} - ${meetingEndTime} ${timeSuffix}`;
 }
+
+export function keep24HourTime(startTime?: HourMinute, endTime?: HourMinute): string | undefined {
+    if (!startTime || !endTime) {
+        return;
+    }
+
+    const formattedStartHour = `${startTime.hour}`;
+    const formattedEndHour = `${endTime.hour}`;
+
+    const formattedStartMinute = `${startTime.minute}`;
+    const formattedEndMinute = `${endTime.minute}`;
+
+    const meetingStartTime = `${formattedStartHour}:${formattedStartMinute.padStart(2, '0')}`;
+    const meetingEndTime = `${formattedEndHour}:${formattedEndMinute.padStart(2, '0')}`;
+
+    return `${meetingStartTime} - ${meetingEndTime}`;
+}
```

Comment:
I think this would be better in a separate file, like `src/lib/time.ts` or something. Also, it would be helpful if you could write tests or JSDoc describing the specs of what it's supposed to do.